domains/slavicman.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "The Slavic Dev Website",
+  "repo": "https://github.com/SlavicMan/atomixdev.github.io",
+  "owner": {
+    "username": "SlavicMan",
+    "email": "kizz4114@yahoo.com"
+  },
+  "record": {
+    "CNAME": "slavicman.github.io/atomixdev.github.io"
```

Comment:
The cname is incorrect. It has to be a hostname. With github the convention is to have `your-github-username.github.io`. I'd suggest you rename your repo to `slavicman.github.io` instead. You can take a look at the [github pages guide](https://guides.github.com/features/pages/) for more info about it.

Reply:
so change the repo name to slavicman.github.io and change the cname record to the same?